examples/randomWalk.js

```diff
@@ -11,13 +11,33 @@ function stepFunc() {
     return pg.geometry.toCircle(2 * Math.PI * Math.random())
 }
 
+function evenCircleStart(numPoints, radius){
```

Comment:
Given we're moving archSpiralStart (rather, spiral) into geometry, this function seems unnecessary.